WalletWasabi.Tests/UnitTests/BlockchainAnalysis/CoinJoinAnonscoreTests.cs

```diff
@@ -239,4 +240,57 @@ public void InputMergePunishmentNoInheritance()
 		// 10 participants, 1 is you, your anonset would be 10 normally and now too:
 		Assert.Equal(10, tx.WalletOutputs.First().HdPubKey.AnonymitySet);
 	}
+
+	public void SiblingCoinjoinDoesntContributeToAnonScore()
```

Comment:
```suggestion

	[Fact]
	public void SiblingCoinjoinDoesntContributeToAnonScore()
```

Reply:
Fixed in https://github.com/zkSNACKs/WalletWasabi/pull/7932/commits/0c9537ceae8420f0bfbfb96ff18bb220f5bfeeb2.